test/e2e/update_retry_on_conflict_test.go

```diff
@@ -191,6 +191,13 @@ spec:
 
 		require.Contains(t, err.Error(), "please apply your changes to the latest version and try again (reason: Conflict)",
 			"Expected error to include k8s reason")
+
+		recalculatedDiff := `Recalculated diff:
+ 43, 43 -   - port: 6380
```

Comment:
Seems like the line numbers are going to be different on different versions of k8s (because of different set of fields getting added). 
I will make some changes to it.